src/__tests__/team.test.js

```diff
@@ -23,45 +22,37 @@ const tasks = [
 
 const team = {};
 
-test.use(config);
 test.describe("Teams", () => {
-  let page;
-
-  test.beforeEach(async ({ browser }) => {
-    const context = await browser.newContext({
-      viewport: {
-        width: 1024,
-        height: 800,
-      },
-    });
-
-    page = await context.newPage();
 
+  test.beforeEach(async ({ page }) => {
     await loadData(page, { tasks, team });
   });
 
-  test("should be editable", async () => {
+  test("should be editable", async ({ page }) => {
     const innerTextBefore = await getTestNameInnerText(page, "Legend-list");
     expect(innerTextBefore).toMatchSnapshot("team-1.txt");
     expect(await page.locator("canvas").screenshot()).toMatchSnapshot(
       "canvas-screenshot-1.png"
     );
 
     // Change team configuration.
-    await setEditorText(
-      page,
-      "team",
-      JSON.stringify({
-        bvaughn: {
-          avatar: "https://avatars.githubusercontent.com/u/29597",
-          name: "Brian",
-        },
-        team: {
-          avatar: null,
-          name: "Unclaimed",
-        },
-      })
-    );
+    await Promise.all([
+      page.waitForResponse("https://avatars.githubusercontent.com/**"),
```

Comment:
It wasn't react update, it was this network request that made you do slomo. Never do timeouts :)

Reply:
Seeing some unexpected weirdness locally from this change:
```
Timeout of 30000ms exceeded.
page.waitForResponse: Page closed
=========================== logs ===========================
waiting for response "https://avatars.githubusercontent.com/**"
============================================================

  37 |     // Change team configuration.
  38 |     await Promise.all([
> 39 |       page.waitForResponse("https://avatars.githubusercontent.com/**"),
     |            ^
  40 |       setEditorText(
  41 |         page,
  42 |         "team",

    at /Users/bvaughn/Documents/git/planner/src/__tests__/team.test.js:39:12
```

Interestingly enough, if I run tests as headfull (no slowMo) they're okay, and if I replace `page.waitForResponse` with `page.waitForRequest` - they're okay to (although the snapshots fail obviously).